skxray/cdi.py

```diff
@@ -96,6 +98,27 @@ def gauss(dims, sigma):
     return y / np.sum(y)
 
 
+def _fft_helper(x, option='fft'):
```

Comment:
I also would not call this 'normed' with the amplitude being scaled by something.